### PR TITLE
Update URL to WDK10.0.17134

### PR DESCRIPTION
**Pull request by OwenSmith**
Updates the download URL and checksum to WDK 10.1.17134.1

Signed-off-by: Owen Smith <owen.smith@citrix.com>

### CODE REVIEW

File: windowsdriverkit10.nuspec [hunk 1]
```diff
@@ -4,10 +4,10 @@
   <metadata>
     <id>windowsdriverkit10</id>
     <title>Windows Driver Kit 10</title>
-    <version>10.0.10586</version>
+    <version>10.1.17134.1</version>
```
Comment (nioncode) [inline windowsdriverkit10.nuspec]: Shouldn't this be 10.**0**.17134.1?

Comment (OwenSmith) [reply windowsdriverkit10.nuspec]: It seems even Microsoft are confused about this.
wdksetup.exe is versioned 10.1.17134.1, which installs WDK 10.0.17134.1 (according to Add/Remove programs) , which also displays version 10.1.17134.1 as its package version...
wdk-version

Comment (nioncode) [reply windowsdriverkit10.nuspec]: How does it show up in Visual Studio? I think this should be the definitive version.

Comment (OwenSmith) [reply windowsdriverkit10.nuspec]: Visual Studio 2017 reports the plugin version 10.0.17110.0
The WDK includes are under a versioned path, using 10.0.17134.0

I'd be inclinded to suggest the version should be 10.0.17134 with no 4th digit if thats an acceptable nuspec entry

Comment (nioncode) [reply windowsdriverkit10.nuspec]: Sounds good. Actually I now see I had the same issue when I first released the package and switched from .1. to .0.

**Comment by OwenSmith** [comment]
Updated PR title to reflect updated versioning

**Comment by OwenSmith** [comment]
Any update on this PR?

**Comment by nioncode** [comment]
Thanks, I merged it just now and will publish it on the chocolatey repo soon.

**Pull request merged**